Projekt/src/app/app.service.ts

```diff
@@ -1,16 +1,16 @@
-import { Injectable } from "@angular/core";
+import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
-import { IDeck } from "./deck.model";
+import { IDeck, IPile } from './deck.model';
 
 @Injectable()
-export class CardsService{
-    constructor(private http: HttpClient){}
+export class CardsService {
+    constructor(private http: HttpClient ) {}
 
-    public getShuffledDeck(){
-        return this.http.get<IDeck>("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1", )
+    public getShuffledDeck(numberOfDecks: number = 0) {
+        return this.http.get<IDeck>('https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1');
```

Comment:
`return this.http.get<IDeck>(`https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=${numberOfCards}`);

Reply:
fixed